### DISCUSSION TITLE
MongoDB source CDC support

### DISCUSSION THREAD

**Discussion by iveselin-glooko** [airbytehq/airbyte#34214] [Connector Questions] [answered]
Hi!
I am posting this question because I am evaluating using Airbyte.
While I was investigating _Sources_ section, I have found that MongoDB source mentioned that it uses Mongo Change Streams and supports CDC - [LINK](https://docs.airbyte.com/integrations/sources/mongodb-v2#cdc).
But under the _Change Data Capture_ section _Current Support_ Mongo is not mentioned - [LINK](https://docs.airbyte.com/understanding-airbyte/cdc#current-support)
Can someone confirm that MongoDB source can be configured to use Change streams for syncing to destination? And if there are any limitations to it?

**Comment by rwask** [comment, marked as answer]
Hi @iveselin-glooko. Thanks for bringing this to our attention! MongoDB is a supported source with CDC.

We have added MongoDB to the supported connectors list for [CDC](https://docs.airbyte.com/understanding-airbyte/cdc). You can read more regarding the implementation with MongoDB on the following article: [Mongodb CDC: How to Sync in Near Real-Time](https://airbyte.com/data-engineering-resources/mongodb-cdc)
